src/data/people.json

```diff
@@ -1,23 +1,23 @@
 [
     {
-        "title": "Taylor Otwell",
-        "known_for": "Creator of Laravel",
+        "title": "Jeffrey Way",
+        "known_for": "Developer of Laravel",
```

Comment:
This is wrong. He is not `Developer of Laravel`. It should be `Creator of Laracasts`.

Reply:
sorry, fixed the typo.